tests/unit_tests/model/instruments/test_crypto_perpetual_pyo3.py

```diff
@@ -32,30 +33,43 @@ def test_hash():
 
 def test_to_dict():
     result = _ETHUSDT_PERP.to_dict()
-    assert CryptoPerpetual.from_dict(result) == _ETHUSDT_PERP
+    assert nautilus_pyo3.CryptoPerpetual.from_dict(result) == _ETHUSDT_PERP
     assert result == {
         "type": "CryptoPerpetual",
         "id": "ETHUSDT-PERP.BINANCE",
-        "raw_symbol": "ETHUSDT",
+        "raw_symbol": "ETHUSDT-PERP",
         "base_currency": "ETH",
         "quote_currency": "USDT",
         "settlement_currency": "USDT",
         "is_inverse": False,
         "price_precision": 2,
-        "size_precision": 0,
+        "size_precision": 3,
         "price_increment": "0.01",
         "size_increment": "0.001",
-        "lot_size": None,
+        "lot_size": "1",
```

Comment:
Is this correct if the size precision is 3? `lot_size` might not be applicable for this instrument?